vms/platformvm/blocks/executor/rejector.go

```diff
@@ -78,6 +77,5 @@ func (r *rejector) rejectBlock(b blocks.Block, blockType string) error {
 		}
 	}
 
-	r.state.AddStatelessBlock(b, choices.Rejected)
 	return r.state.Commit()
```

Comment:
We aren't committing anything here anymore - so I think we can remove this as well.